alphapept/__init__.py

```diff
@@ -1 +1,7 @@
 __version__ = "0.3.30"
+
+__requirements__ = {
+    "": "requirements/requirements.txt",
+    "develop": "requirements/requirements_develop.txt",
+    "gui": "requirements/requirements_gui.txt",
+}
```

Comment:
This is not the place to comment on it, but I ahve been thinking this is a redundant step if we are consistent enough. If we adhere to a scheme where all requirements are always in the requirement folder and we always use `requirements-xxx.txt`, this setup in the `__init__.py` can be omitted. (This comment is more aimed towards the alphatemplate though, but here is a probably a good use case)